integrationtests/tests/fixtures/mod.rs

```diff
@@ -1210,18 +1262,11 @@ impl FederationTest {
     }
 }
 
-pub fn assert_ci<M: ModuleConsensusItem>(ci: &ConsensusItem) -> &M {
-    if let ConsensusItem::Module(mci) = ci {
-        assert_module_ci(mci)
-    } else {
-        panic!("Not a module consensus item");
-    }
-}
-
-pub fn assert_module_ci<M: ModuleConsensusItem>(mci: &PerModuleConsensusItem) -> &M {
-    debug!(
-        module_instance_id = mci.module_instance_id(),
-        "Checking module consensus item"
-    );
-    mci.as_any().downcast_ref().unwrap()
+/// Unwraps a dyn consensus item into a specific one for making assertions
+pub fn unwrap_item<M: ModuleConsensusItem>(mci: &Option<DynModuleConsensusItem>) -> &M {
+    mci.as_ref()
+        .expect("Module item exists")
```

Comment:
If you require it to exist, why take `Option`, and not just `&DynModuleConsensusItem`.

Reply:
It's really a convenience method for the results of `find_module_item`